src/main/java/com/clevertap/stormdb/maps/IndexMap.java

```diff
@@ -26,4 +26,11 @@
      * @return Size of the index.
      */
     int size();
+
+    /**
+     * API to support key deletion from the map
+     * @param key The key to be deleted
+     * @return
+     */
+    int remove(int key);
```

Comment:
What are we returning?

Reply:
TIntIntHashmap returns an int value itself while removing the key so I thought of keeping the same functionality in the indexMap (defaultIndexMap) also.

Reply:
It returns the value just before the remove operation. Can we add some Javadoc?